pkgs/top-level/all-packages.nix

```diff
@@ -10835,6 +10835,10 @@ with pkgs;
 
   nginxModules = callPackage ../servers/http/nginx/modules.nix { };
 
+  nginxShibboleth = callPackage ../servers/http/nginx/stable.nix {
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders nginxModules.shibboleth ];
+  };
```

Comment:
@Mic92 another possibility is to just add this with a comment above it saying that if these particular compositions start proliferating, we should look into the loadable modules. I'm obviously partial to not having to compile this myself on every machine I use it on, "just this one time..." 😄